config/Version.xcconfig

```diff
@@ -1,2 +1,2 @@
-VERSION_LONG = 7.77.0.3
+VERSION_LONG = 7.77.0.4
```

Comment:
Bumped to test the new set up. See https://buildkite.com/automattic/pocket-casts-ios/builds/8883.